src/pages/Admin/Venue/QuestionInput.tsx

```diff
@@ -66,7 +68,10 @@ const QuestionInput: React.FC<QuestionInputProps> = ({
   };
 
   return (
-    <div className="input-container" style={{ marginBottom: "1.5rem" }}>
+    <div
+      className={`input-container ${className}`}
+      style={{ marginBottom: "1.5rem" }}
+    >
```

Comment:
Similar to `EntranceInput`, an added class following BEM can help clear out the inline style.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2012/commits/a404f57bd688146d8e490c8728fbf54689a7fafc